letsencrypt/renewal.py

```diff
@@ -102,16 +102,14 @@ def _restore_webroot_config(config, renewalparams):
     form.
     """
     if "webroot_map" in renewalparams:
-        # if the user does anything that would create a new webroot map on the
-        # CLI, don't use the old one
-        if not (cli.set_by_cli("webroot_map") or cli.set_by_cli("webroot_path")):
-            setattr(config.namespace, "webroot_map", renewalparams["webroot_map"])
+        if not cli.set_by_cli("webroot_map"):
```

Comment:
I think the previous semantics might be less confusing to people here.  Suppose on day 1 you did:
```
letsencrypt certonly --webroot -w /var/www/ -d example.com,www.example.com
```
And then you did:
```
letsencrypt renew -w /var/www/moved
```
I believe the change you've made here would cause the webroot to remain `/var/www/` during renewal, because the client would construct and save a `webroot_map` during first execution.